sites/bizbash/components/directory-search/index.marko

```diff
@@ -16,66 +16,76 @@ $ const searchParams = {
   pageNumber: page,
   size: input.pageSize,
   phrase,
+  minimumCapacity,
   facets: [
     { field: 'market', value: marketId },
     { field: 'topic', value: topicId },
   ]
 };
-
-<div class="row">
-  <div class="col mb-block">
-    <form class="directory-search">
-      <input name="phrase" type="search" class="form-control directory-search__input" placeholder="Enter search phrase..." value=phrase>
-      <if(marketId)><input name="marketId" type="hidden" value=marketId></if>
-      <if(topicId)><input name="topicId" type="hidden" value=topicId></if>
-      <button type="submit" class="btn btn-primary directory-search__button">Search</button>
-    </form>
+<form id="directory-search">
+  <div class="row">
+    <div class="col mb-block">
+      <div class="directory-search">
+        <input form="directory-search" name="phrase" type="search" class="form-control directory-search__input" placeholder="Enter search phrase..." value=phrase>
+        <if(marketId)><input form="directory-search" name="marketId" type="hidden" value=marketId></if>
+        <if(topicId)><input form="directory-search" name="topicId" type="hidden" value=topicId></if>
+        <button type="submit" class="btn btn-primary directory-search__button">Search</button>
+      </div>
+    </div>
   </div>
-</div>
-<div class="row">
-  <div class="col-md-5 col-lg-4 mb-block">
-    <div class="directory-facets-toggle">
-      <div class="directory-facets-toggle__label">Show Filters</div>
-      <cms-browser-component
-        name="MenuToggleButton"
-        props={
-          className: "directory-facets-toggle__button",
-          targets: [".directory-facets"],
-          toggleClass: "directory-facets--open",
-          iconModifiers: ["sm"],
-          iconName: "plus",
-          expandedIconName: "dash",
-        }
+  <div class="row">
+    <div class="col-md-5 col-lg-4 mb-block">
+      <div class="directory-facets-toggle">
+        <div class="directory-facets-toggle__label">Show Filters</div>
+        <cms-browser-component
+          name="MenuToggleButton"
+          props={
+            className: "directory-facets-toggle__button",
+            targets: [".directory-facets"],
+            toggleClass: "directory-facets--open",
+            iconModifiers: ["sm"],
+            iconName: "plus",
+            expandedIconName: "dash",
+          }
+        />
+      </div>
+      <if(input.contentType === 'Venue')>
+        <div class="directory-facets">
+          <h3 class="directory-facets__title">Venue Capacity</h3>
+          <div class="input-wrapper">
+            <input form="directory-search" name="minimumCapacity" type="number" class="form-control directory-search__input" placeholder="Enter Minimun Capacity..." value=minimumCapacity>
```

Comment:
How would someone easily clear this field after a value is set? i.e. an "X" or "clear" button?

Reply:
@zarathustra323 

- form wrapper is because of the inputs being disconnected
- the css should be update
- form attribute is removed
- minimumCapacity doesn't need to be hidden due to it being part of the form
- clear function you can highlight and delete any value and hit enter.  Adding a clear button would reset the whole form.  

Thoughts?